### PR TITLE
Added PD DB W1

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: DB
Week(s): 1

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Added PD DB W1

## Who needs to know about this?

@kfklein15 @CodeYourFuture/syllabus-team

### CODE REVIEW

File: docs/db/week-1/homework.md [hunk 1]
```diff
@@ -37,3 +37,7 @@ https://docs.codeyourfuture.io/course-processes/running-the-course/codewars#code
 Every week you need to complete at least three kata. Spend at least 20 minutes, three times a week, on your kata workout.
 
 Find the collection on the CodeYourFuture account: https://www.codewars.com/users/CodeYourFuture/authored_collections
+
+## 5) STAR Framework  (PD - 4 hr)
+
+Use [the provided spreadsheet](https://docs.google.com/spreadsheets/d/1JvPFFewdrVP7nY7sL7l13TeGUxLos7h3NxWb0itxm84/edit#gid=0) to come up with interview examples in line with the STAR framework.
```
Comment (kfklein15) [inline docs/db/week-1/homework.md]:
```suggestion
Use [the provided spreadsheet](https://docs.google.com/spreadsheets/d/1JvPFFewdrVP7nY7sL7l13TeGUxLos7h3NxWb0itxm84/edit#gid=0) to come up with interview examples in line with the STAR framework.
Don't forget to make a copy of this spreadsheet, before adding your information to it.
```

File: docs/db/week-1/lesson.md [hunk 2]
```diff
@@ -299,6 +300,34 @@ SELECT * FROM hotels WHERE postcode = 'CM194JS' OR postcode = 'TR209AX';
...
 :::
 
+## PD (Competency-based interviews)
+**Session objective:** This session explores the competency-based interview concept, and the STAR framework.
+
+[Competency-based interviews slides](https://docs.google.com/presentation/d/1YXPYKanQjfFyegqPaY0h_f3l-ojceZxcFr4cGjCqbaM/edit#slide=id.ga9333c68a9_0_33)
+
+:::note Exercise (5 min)
+
+**Exercise objective:** To observe implementation of STAR framework for competency-based interviews
```
Comment (kfklein15) [inline docs/db/week-1/lesson.md]: Do we need both exercises? Just asking since it is more effective to analyse than observe.

**Review by kfklein15 — CHANGES_REQUESTED** [review]

File: docs/db/week-1/lesson.md (hunk 2 above)
Comment (esma-g) [reply docs/db/week-1/lesson.md]: They are not only observing but also discussing following their observation.

**Comment by esma-g** [comment]
@illicitonion this PR has conflicting files. I have no idea how to resolve them, can you help please?

**Pull request closed without merging**